libs/librepcb/project/circuit/netsignal.cpp

```diff
@@ -263,7 +265,13 @@ void NetSignal::updateErcMessages() noexcept {
     mErcMsgUnusedNetSignal.reset();
   }
 
-  if (mIsAddedToCircuit && (mRegisteredComponentSignals.count() < 2)) {
+  int registeredRealComponentCount = 0;//mRegisteredComponentSignals.count();
```

Comment:
Instead of commented out code, I'd prefer a short comment why not all component signals are counted :wink:

```suggestion
  // Raise a warning if the net signal is connected to less then two component
  // signals. But do not count component signals of schematic-only components
  // since these are just "virtual" connections, i.e. not represented by a
  // real pad (see https://github.com/LibrePCB/LibrePCB/issues/739).
  int registeredRealComponentCount = 0;
```